src/openloco/windows/townwnd.cpp

```diff
@@ -131,24 +132,19 @@ namespace openloco::ui::windows::town
             self->draw(dpi);
             common::drawTabs(self, dpi);
             self->drawViewports(dpi);
-            widget::drawViewportCentreButton(dpi, self, (widget_index)widx::centre_on_viewport);
-
-            static string_id townSizeLabelIds[] = {
-                string_ids::town_size_hamlet,
-                string_ids::town_size_village,
-                string_ids::town_size_town,
-                string_ids::town_size_city,
-                string_ids::town_size_metropolis,
-            };
+            widget::drawViewportCentreButton(dpi, self, widx::centre_on_viewport);
 
             auto town = townmgr::get(self->number);
-            commonFormatArgs[0] = townSizeLabelIds[(uint8_t)town->size];
-            *(int32_t*)&commonFormatArgs[1] = town->population;
 
-            const uint16_t xPos = self->x + self->widgets[widx::unk_8].left - 1;
-            const uint16_t yPos = self->y + self->widgets[widx::unk_8].top - 1;
-            const uint16_t width = self->widgets[widx::unk_8].width() - 1;
-            gfx::draw_string_494BBF(*dpi, xPos, yPos, width, colour::black, string_ids::status_town_population, &*commonFormatArgs);
+            auto args = FormatArguments();
+            args.push(town->getTownSizeString());
+            args.push(town->population);
+
+            const auto& widget = self->widgets[widx::unk_8];
```

Comment:
You forgot to name `unk_8`!!!!1!

Reply:
what did we decide it was again? The viewport frame?

Reply:
It's the status bar / resize handle.